ext/devcontainer/src/azd/README.md

```diff
@@ -0,0 +1,19 @@
+# Azure Developer CLI (azure-dev-cli)
+
+Installs the [Azure Developer CLI](https://github.com/Azure/azure-dev) along with needed dependencies.
+
+## Example Usage
+
+```json
+"features": {
+        "ghcr.io/azure/azure-dev/azd:1": {
```

Comment:
spacing feels inconcistent

Reply:
Sloppy formatting because it's not "really" JSON when it got pasted.